pytorch_blade/pytorch_blade/compiler/jit/torch/shape_analysis.cpp

```diff
@@ -933,8 +933,9 @@ class ShapePropagator : public PropertyPropBase {
     //   - First input should be the only tensor input
     static const register_formula_for simple_unary_ops{
         {
-            "aten::t(Tensor self) -> Tensor",
+            "aten::narrow(Tensor self, int dim, int start, int length) -> Tensor",
```

Comment:
why `narrow` is moved here? It should have same input/output scalar type.

Reply:
narrow preserved the rank but change the dimension size.